### ISSUE TITLE
Xendit WebView crash in iOS 11.1.2

### ISSUE THREAD

**Issue by datamaru** [xendit/xendit-sdk-ios#5]
Error text : 
Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[<Xendit.WebViewController 0x143fccd70> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key containerView.'

I'm using Xendit 1.2.0 and iOS 11.1.2

**Comment by afik** [comment]
Hi @datamaru thanks for submitting the issue. Could you tell us step by step that you did when you encountered this issue?

**Comment by cherylchua** [comment]
Hi @afik, from the WA chat, it seems that after entering the credit card information, the app crashed while waiting for the 3ds page.

**Comment by afik** [comment]
Hey @datamaru, did you encountered this error using our sample apps or using your own application? 

If you're using your own apps, have you passed `UIViewController` on `createAuthentication` method? You can check the example in https://github.com/xendit/xendit-sdk-ios/blob/master/XenditExample/XenditExample/Controllers/CreateAuthentification.swift#L44

**Comment by datamaru** [comment]
I'm using Xendit.createToken in my own application. It's still works fine when we tested it on iOS 10.3 but crashed on 11.1

Here's the code : 
Xendit.createToken(fromViewController: self, cardData: cardData) { (token, error) in
            if (error != nil) {
                self.dismissLoadScreen()
                // Handle error. Error is of type XenditError
                AlertManager.alertWithHandler(viewController: self, title: (error?.errorCode)!, message: (error?.message)!, handler: { _ in
                    self.dismiss(animated: true, completion: nil)
                })
                return
            }
            self.dismissLoadScreen()
            self.xenditCCToken = token!.id
            self.status = token!.status
            
            print("Xendit create token response: ")
            print(self.xenditCCToken)
            print(self.status)
            
            self.xenditCCCharge()
        }
    }

**Comment by 0505gonzalez** [comment]
Hi @datamaru, does `self` in your code refer to a UIViewController, as in our example? https://github.com/xendit/xendit-sdk-ios/blob/master/XenditExample/XenditExample/Controllers/CreateTokenViewController.swift#L13

**Comment by datamaru** [comment]
Yes, it does refer to UIViewController. I never changed the code from before updating to iOS 11.1 and in iOS 10 it works fine.

**Comment by afik** [comment]
@datamaru we still cannot reproduce the issue from our side. We can run this code https://github.com/xendit/xendit-sdk-ios/blob/repro-issue/XenditExample/XenditExample/Controllers/CreateTokenViewController.swift using iOS 11.1 and Xendit Xendit 1.2.0 without any errors. 

Could you try to run this example apps and compare it to your own code?
